libflux/flux/FLUXDOC.md

```diff
@@ -354,6 +354,7 @@ enum Doc {
 }
 
 struct PackageDoc {
+       path: String,
 	name: String,
```

Comment:
Can we fix the formatting here?

Reply:
Also, how are we populating the path when parsing? Is this field just going unused for now?

Reply:
Just realized this is just the example code in the spec. I would prefer if changes to the spec were accompanied by the appropriate changes to the code. Can we include that as part of this PR?

Reply:
What exactly is wrong with the formatting? Check out the most recent parser changes, but basically we are using the PolyTypeMap to pass in the relative path when populating.

Reply:
Just the indentation. The fields in a struct should all be aligned.

Did we already do the work to include the path in `PackageDoc`? If so, then disregard my last comment.

Reply:
Does this look better, it is rendering very weird on my device